modules/codegen/src/smithy4s/codegen/internals/CollisionAvoidance.scala

```diff
@@ -279,7 +279,30 @@ private[internals] object CollisionAvoidance {
     "notify",
     "notifyAll",
     "toString",
-    "wait"
+    "wait",
+
+    // scala 3 "regular" keywords
+    // https://docs.scala-lang.org/scala3/guides/migration/incompat-syntactic.html#restricted-keywords
+    "enum",
+    "export",
+    "given",
+    "then",
+
+    // scala 3 "soft" keywords
+    // https://dotty.epfl.ch/docs/internals/syntax.html#soft-keywords-3
+    // https://docs.scala-lang.org/scala3/reference/soft-modifier.html
+    "as",
+    "derives",
+    "end",
+    "erased",
+    "extension",
+    "infix",
+    "inline",
+    "opaque",
+    "open",
+    "throws",
+    "transparent",
+    "using",
```

Comment:
the soft keywords are *probably* not needed in this list, and I'm guessing they furthermore don't belong. still, best that y'all are aware of the distinct keyword types so you can make the correct decision.

Reply:
I'm thinking if they don't produce a listing issue, they can stay ignored for now. Let's see what other maintainers think.

Reply:
I agree with Jakub

Reply:
> I'm thinking if they don't produce a listing issue, 

Linting* 😅

Reply:
removed them